src/main/java/fi/metatavu/soteapi/content/ContentController.java

```diff
@@ -91,12 +92,14 @@ public ContentData createContentData(String language, String value, Content cont
    * @param modifier modifier
    * @return updated content
    */
-  public Content updateContent(Content content, String originId, String slug, Content parent, ContentType contentType, String category, Long orderIndex) {
+  public Content updateContent(Content content, String originId, String slug, Content parent, ContentType contentType, String category, OffsetDateTime created, OffsetDateTime modified, Long orderIndex) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Method has 9 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)